browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -2979,10 +2979,10 @@ If you don't accept this request, VPN will not reconnect and your internet conne
       Network
     </message>
     <message name="IDS_BRAVE_WALLET_ALLOW_ADD_NETWORK_DESCRIPTION" desc="Add network panel description">
-      This will allow this network to be used within Brave Wallet.
+      This will allow this network to be used within Brave Wallet. Learn more.
     </message>
     <message name="IDS_BRAVE_WALLET" desc="Brave Wallet text for string detection with localization">
```

Comment:
nit: The name and description should be updated as per the updated text value.

Reply:
I updated it, but build was not successful, there was an error about local, `'string/learn_more' has a conflicting value for configuration (ta).`

Reply:
it sounds like a duplicate resource error or something, a quick fix would be to use a different name like `IDS_BRAVE_WALLET_LEARN_MORE`. Use slack with the complete error log if there's no quick fix. (though it's a minor thing so leave it if it takes too much time, I will look into it later)